src/board_controller/openbci/ganglion.cpp

```diff
@@ -479,6 +396,138 @@ void Ganglion::read_thread ()
     delete[] package;
 }
 
+void Ganglion::decompress_firmware_3 (
+
```

Comment:
strange that clang format does not remote it

Reply:
I tried to make it one line, but the clang-format settings move the parameters to the second line. I think it may be because the line is longer than the max line length. Is there a different format you'd like this in?

Reply:
Oh weird - should be fixed now.